src/views/partials/product-card.ejs

```diff
@@ -0,0 +1,44 @@
+<section class="card-wrapper">
+    <div class="card">
+        <div class="product-head">
+            <div class="line"></div>
+            <p class="product-price"> $<%= product.product_price %></p>
+            <a href="#">
+                <h6 class="product-name"><%= product.product_name %></h6>
+            </a>
+            <div class="ratings-review">
+                <div class="ratings">
+                    <i class="faz fa-star"></i>
+                    <i class="faz fa-star"></i>
+                    <i class="faz fa-star"></i>       
+                    <i class="faz fa-star"></i>
+                    <i class="faz fa-star"></i>
+                </div>
+                <a class="review" href="#">Write A Review</a>      
+            </div>
+        </div>
+        <p class="in-stock">
+            <i class="fas fa-circle" class="circle"></i>
+         In stock
+        </p>
+    </div>
+    <div class="description-wrapper">
+        <p class="description"><%= product.product_description %></p>
+    </div>
+    <form class="add-cart">
+        <div class="add-btn">
+            <p class="qty">Qty</p>
+            <div class="quantity">
+                <input type="number" class="qty-number" id="qty" step="1" min="1" max="300" name="quantity" value="1">
```

Comment:
the max value could be product.product_in_stock, therefore the user can never pick more than there is